cli/cmd/initialize/init.go

```diff
@@ -36,6 +38,33 @@ func InitCmd(cfg *config.Config) *cobra.Command {
 
 					projectPath = repoName
 				}
+			} else {
+				if mode {
+					return fmt.Errorf("please provide the project path, please run 'rill start <project-path>' ")
+				}
+
```

Comment:
Print `"No existing project found. Enter name to initialize a new Rill project."`